build.gradle

```diff
@@ -2116,6 +2109,17 @@ project(':streams:streams-scala') {
     dependsOn 'copyDependantLibs'
   }
 
+  // spotless 6.14 requires Java 11 at runtime
+  if (JavaVersion.current().isJava11Compatible()) {
```

Comment:
Does it imply the developers who want to contribute code to streams-scala must use JDK11 as our Ci will check that?

Reply:
CI will check it.

Reply:
I mean, developers don't have to use Java 11. They can rely on CI for the rare case where this module is being updated and they didn't follow the style rules. It is more practical to use Java 11 or newer though. I don't think it's a problem - Java 8 support is more for deployments than for contributors.